thousands/auth.py

```diff
@@ -60,6 +60,18 @@ def vk_login():
     return oauth_login(request, vk_flow('state'), vk_get_user)
 
 
+@app.route('/login/as/<user_id>')
+def login_as(user_id):
+    if app.config['TESTING']:
```

Comment:
I think, we can use `app.config['DEBUG']` instead http://flask.pocoo.org/docs/0.10/config/